etna/core/blocks/quote.py

```diff
@@ -8,7 +8,6 @@ class QuoteBlock(blocks.StructBlock):
     Quote streamfield component
     """
 
-    heading = blocks.CharBlock(required=False, max_length=100)
```

Comment:
Task: The FE still has this field there. Can you remove it there, double check the layout isn't broken because of the change?